### ISSUE TITLE
`app.UseHttpsRedirection()` behind  IIS 

### ISSUE THREAD

**Issue by John0King** [aspnet/BasicMiddleware#352]
Will ANCM automatice  set `ASPNETCORE_HTTPS_PORT`  environment variable ?

**Comment by Tratcher** [comment]
@shirhatti that's a good idea.

**Comment by Tratcher** [comment]
@John0King are you running IIS on 443 or a custom port?

**Comment by shirhatti** [comment]
Not sure I understand the ask here. Do you want ANCM to enforce HTTPS redirection?

**Comment by John0King** [comment]
@Tratcher  port 443.

------------------

IIS Express with Visual Studio will set `ASPNETCORE_HTTPS_PORT` environment variable (so the default template work) ,  but running on IIS do not have this environment variable.

**Comment by jkotalik** [comment]
Yeah I don't think there is a good way to reliably redirect to Https with IIS easily.
